integr8ly/integreatly-operator
# MGDAPI-4326 Test case updates for RHOAM v1.24.0

# Issue link


https://issues.redhat.com/browse/MGDAPI-4326

# What


Standard test case updates.
Fix of prometheus route name (used to be "prometheus-route", now it is just "prometheus"

# Verification steps




Eye review only should suffice.
You can optionally verify that all the `1.24.0` test cases were indeed generated for RHOAM v1.24.0 and try to run that ./scripts/alerts-during-perf-testing.sh against existing cluster to see that it indeed works.
